musicAlbum/music_album_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'music_album'
+
+describe MusicAlbum do
+  before :each do
+    @music_album = MusicAlbum.new('2020-02-02', on_spotify: true)
+  end
+
+  it 'Test for MusicAlbum instance' do
+    expect(@music_album).to be_instance_of MusicAlbum
+  end
+
+  it 'Test for can be archive method' do
+    expect(@music_album.send('can_be_archived?')).to be false
+  end
```

Comment:
- [ ] This test is failing. Please try to make it green ✔️ 

![image](https://github.com/MuhammadUsmanAslam/catalog_of_my_things/assets/46839548/b6b2b98a-78e1-4f95-b3fc-3b372b69b8b6)